src/test/java/com/wikia/webdriver/testcases/communitypagetests/SpecialCommunityPageTests.java

```diff
@@ -52,7 +54,7 @@ public void verifyUserEditingFlow() {
 
   private String getEditUrl(String page, String query) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "getEditUrl" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)